apps/web/screens/UniversitySearch/UniversitySearch.tsx

```diff
@@ -0,0 +1,1219 @@
+                                />
+                              }
+                              onCheck={(e) =>
+                                handleComparisonChange({
+                                  id: dataItem.id,
+                                  nameIs:
+                                    locale === 'en'
+                                      ? dataItem.nameEn
+                                      : dataItem.nameIs,
+                                  iconSrc: universities.filter(
+                                    (x) => x.id === dataItem.universityId,
+                                  )[0].logoUrl,
+                                })
+                              }
+                              checked={
+                                selectedComparison.filter(
+                                  (x) => x.id === dataItem.id,
+                                ).length > 0
+                              }
+                              buttonLabel={n('apply', 'Sækja um')}
+                              checkboxLabel={n('compare', 'Setja í samanburð')}
+                              checkboxId={dataItem.id}
+                              cta={createPrimaryCTA(dataItem)}
+                              href={
+                                locale === 'en'
+                                  ? `/en/haskolanam/${dataItem.id}`
+                                  : `/haskolanam/${dataItem.id}`
+                              }
+                              infoItems={[
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'school'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n(
+                                    dataItem.degreeType,
+                                    TranslationDefaults[dataItem.degreeType],
+                                  )}`,
+                                },
+                                {
+                                  icon: (
+                                    <Icon
+                                      icon={'calendar'}
+                                      type="outline"
+                                      color="blue400"
+                                    />
+                                  ),
+                                  title: `${n('begins', 'Hefst')} ${n(
+                                    dataItem.startingSemesterSeason,
+                                    TranslationDefaults[
+                                      dataItem.startingSemesterSeason
+                                    ],
+                                  )} ${dataItem.startingSemesterYear}`,
+                                },
+                                // {
+                                //   icon: (
```

Comment:
Can this commented code be removed, will it be re-introduced later?

Reply:
Yes it is part of the new data that will be coming soon